runtime/runtime.m

```diff
@@ -146,6 +146,12 @@
 	const char *EntryAssemblyPath;
 #endif
 	struct AssemblyLocations* AssemblyLocations;
+#if DOTNET
```

Comment:
why not use the same condition as below ?
> #if defined (CORECLR_RUNTIME)
> #if !defined(__aarch64__)

Reply:
maybe not the 2nd one as the struct would not be identical in managed land (at least it would be harder than it needs to be)

Reply:
Yes, that was exactly my thinking: this struct must match the corresponding struct in managed land, and there we use the same managed code for MonoVM and CoreCLR.

Reply:
> Maybe add a comment ? about same struct on both VM (for dotnet)

Will do.

> I assume there's already one (not seen in the diff) about matching struct in the managed land

Correct:

https://github.com/xamarin/xamarin-macios/blob/4d6ab1d6c8ae555ec819efc94fa44d2c3aa0fa5d/runtime/runtime.m#L99